### PR TITLE
feat!: throw error if insufficient number of points

### PR DESCRIPTION
**Pull request by EscapedGibbon**
close: https://github.com/mljs/regression/issues/34

### CODE REVIEW

File: src/regression/poly-fit-regression2d.js [hunk 1]
```diff
@@ -66,29 +66,19 @@ export default class PolynomialFitRegression2D extends BaseRegression {
 
     var examples = X.rows;
     var coefficients = ((this.order + 2) * (this.order + 1)) / 2;
+    if (examples < coefficients) {
+      throw new Error(
+        'Insufficient number of points to create regression model.',
```
Comment (stropitek) [inline src/regression/poly-fit-regression2d.js:71]: Would give the minimum number of expected points for the model in the error message.

**Comment by targos** [comment]
I don't understand. Why is there already a release v6.0.0 with a fake change that has this commit message? https://github.com/mljs/regression/commit/00eb422f371914643700a0af0efd8ed9c7ef679b

File: src/regression/poly-fit-regression2d.js (hunk 1 above)
Comment (lpatiny) [reply src/regression/poly-fit-regression2d.js:71]: It depends how the points are independent so it depends. This is why we didn't put the number.

**Comment by lpatiny** [comment]
> I don't understand. Why is there already a release v6.0.0 with a fake change that has this commit message? [00eb422](https://github.com/mljs/regression/commit/00eb422f371914643700a0af0efd8ed9c7ef679b)

Yes this is my mistake. I was convinced to have merged this PR before updating the project and adding the github action workflows.

**Pull request closed without merging**